packages/core/lib/configAdapter.js

```diff
@@ -53,25 +47,21 @@ function configureManagedGanache(config, networkConfig, mnemonic) {
     100 // Use as default ether balance for each account
   );
 
-  const blockTime = getFirstDefinedValue(
-    networkConfig.blockTime,
-    0 // Use as default block time
-  );
+  const blockTime = getFirstDefinedValue(networkConfig.blockTime);
```

Comment:
These lines of code, where `getFirstDefinedValue` is called on a single argument with no default, don't make sense.  If you only have a single input and no defaults, there's no need for the function; you can just write `const blockTime = networkConfig.blockTime`, etc.  Also, is there a reason these defaults were deleted?

Reply:
Sure, @haltman-at . In my opinion, these values can be left undefined as we can rely on ganache's default for these values.

Reply:
Makes sense!  Do you know whether those defaults are the same as what we have here, though?  If they're different, it's possible people might be relying on the defaults here.  Indeed even if they are the same, it might make sense to leave in the defaults in case Ganache ever changes them (so that our defaults don't change).

Regardless, if you do take out the defaults, you should also take out the function call.

Reply:
The preference to rely on Ganache's defaults, with exceptions, should be documented in the code

Reply:
> Makes sense! Do you know whether those defaults are the same as what we have here, though? If they're different, it's possible people might be relying on the defaults here. Indeed even if they are the same, it might make sense to leave in the defaults in case Ganache ever changes them (so that our defaults don't change).
> 
> Regardless, if you do take out the defaults, you should also take out the function call.

I will make the changes you suggested @haltman-at . I thought of keeping the scope of adding the defaults again after the discussion :) The `blockTime` and `gasPrice` are same as Ganache's defaults. So, what should be desired behavior for this, should we rely on Ganache's defaults or should we keep our defaults? Any suggestions on this @gnidan @eggplantzzz ?

Reply:
IMO we should probably leave the defaults as-is in this PR, and we can change or remove them later if we think that's warranted?  Just in the interest of keeping the scope of this PR from expanding...

Reply:
I agree with you @haltman-at . We can change them sometime later if desired.